src/option_parser.c

```diff
@@ -250,22 +233,46 @@ int load_ini_file(FILE * fp)
 
                 *equal = '\0';
                 char *key = g_strstrip(start);
-                char *value = g_strstrip(equal + 1);
-
-                char *quote = strchr(value, '"');
-                if (quote) {
-                        char *closing_quote = strchr(quote + 1, '"');
-                        if (!closing_quote) {
-                                fprintf(stderr,
-                                     "Warning: invalid config file at line %d\n",
-                                     line_num);
-                                fprintf(stderr, "Missing '\"'\n");
-                                continue;
+                char *unparsed = equal + 1;
+                char *value = unparsed;
+
+                bool in_quote = false;
+                while ((unparsed = strpbrk(unparsed, "\"\\#;")) != NULL) {
+                        switch (*unparsed) {
+                        case '"':
+                                memmove(unparsed, unparsed + 1, strlen(unparsed));
+                                in_quote = !in_quote;
+                                break;
+                        case '\\':
+                                switch(unparsed[1]) {
+                                case '\\':
+                                case '"':
+                                        memmove(unparsed, unparsed + 1, strlen(unparsed));
+                                        unparsed++;
+                                        break;
+                                default:
+                                        // Unrecognized backslash sequence;
+                                        // skip over it without removing the backslash.
+                                        // Consider issuing an error or warning here instead.
+                                        unparsed += 2;
```

Comment:
Oh! You are of course correct.  I was hasty when gutting out the initial version that considered this case to be an error.